### PR TITLE
Feature/ui catalog category

### PR DESCRIPTION
**Pull request by pkvarnwal**

### CODE REVIEW

File: app/src/main/java/com/eqdepot/adapters/CatalogAdapter.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import butterknife.ButterKnife;
+
+public class CatalogAdapter extends RecyclerView.Adapter<CatalogAdapter.CatalogViewHolder> {
+
+    private LayoutInflater mLayoutInflator;
+    private Activity mActivity;
+    private ArrayList<String> mCatalogs;
+
+    public CatalogAdapter(Activity activity, ArrayList<String> catalogs) {
+        this.mActivity = activity;
```
Comment (ashutoshsingh33) [inline app/src/main/java/com/eqdepot/adapters/CatalogAdapter.java]: no need of this here.

Comment (pkvarnwal) [reply app/src/main/java/com/eqdepot/adapters/CatalogAdapter.java]: Okay

**Comment by ashutoshsingh33** [comment]
@aashishdhawan  Please review.

File: app/src/main/java/com/eqdepot/fragments/catalog/CatalogFragment.java [hunk 2]
```diff
@@ -27,9 +37,22 @@ public static CatalogFragment newInstance() {
...
         ButterKnife.bind(this, view);
-
         ((CatalogActivity) getActivity()).setTitle(getActivity().getResources().getString(R.string.catalog));
 
+        setRecycleAdapter();
+
         return view;
     }
+
+    private void setRecycleAdapter() {
```
Comment (aashishdhawan) [inline app/src/main/java/com/eqdepot/fragments/catalog/CatalogFragment.java]: Please add a TODO

**Pull request merged**